lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -513,80 +553,82 @@ def _get_gps_status_as_bool(enodeb: EnodebAcsStateMachine) -> bool:
 
 
 def _get_and_cache_gps_coords(enodeb: EnodebAcsStateMachine) -> Tuple[
-    str, str,
+    str, str, str,
```

Comment:
since this tuple is growing, I would recommend using `NamedTuple` to create a `GPSTuple` and use that instead

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812058-8f0de79c-db6f-4329-bf88-68c2d2bdd49a.png)
![image](https://user-images.githubusercontent.com/87301024/147812114-da955a74-bc9a-493b-bfe8-24379a79462a.png)
modified thank you !